statistics/selectivity.go

```diff
@@ -240,6 +243,9 @@ func (coll *HistColl) Selectivity(ctx sessionctx.Context, exprs []expression.Exp
 	for _, set := range usedSets {
 		mask &^= set.mask
 		ret *= set.Selectivity
+		if set.partCover {
+			ret *= selectionFactor
```

Comment:
could you add a comment to explain why we multiple `selectionFactor`?

Reply:
Done.